crepecake/src/main/java/com/google/cloud/tools/crepecake/blob/Blob.java

```diff
@@ -29,4 +31,11 @@
    * @return the {@link BlobDescriptor} of the written BLOB
    */
   BlobDescriptor writeTo(OutputStream outputStream) throws IOException;
+
+  /** Writes the BLOB to a string. */
+  default String writeToString() throws IOException {
```

Comment:
I don't think we want to use default here. While it's nice, I think it throws us back into the abstract class pattern. Can this just part of Blobs?